### DISCUSSION TITLE
Template repo, enable read and write workflow permissions

### DISCUSSION THREAD

**Discussion by garronej** [community/community#26654] [Actions]
<p>Hello,<br>
Since recently Workflow permissions have been introduced.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/6/d6b044298144be8e4ab9261eddf636b74fd40968.png" data-download-href="/uploads/short-url/uDdPiuSCorcD3W5UIW58kxwOqI8.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/6702424/181099079-240761d7-fad9-4bd9-bc86-b4cf4604bd3a.png" alt="image" data-base62-sha1="uDdPiuSCorcD3W5UIW58kxwOqI8" width="690" height="182" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d6b044298144be8e4ab9261eddf636b74fd40968_2_690x182.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d6b044298144be8e4ab9261eddf636b74fd40968_2_1035x273.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d6b044298144be8e4ab9261eddf636b74fd40968_2_1380x364.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d6b044298144be8e4ab9261eddf636b74fd40968_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1622×430 60.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I maintain <a href="https://github.com/garronej/ts-ci" rel="noopener nofollow ugc">a public template</a> that automatically setup<br>
the repo for users when they click on<br>
<img src="https://user-images.githubusercontent.com/6702424/181099080-5446ca6c-8c66-4dc0-9f43-0fcfbea5b598.png" alt="image" data-base62-sha1="uI1ULlCMAGex9V0IUYzdO6A6UuS" width="342" height="120"></p>
<p>It would be wonderful if I could configure that the repo generated from <a href="https://github.com/garronej/ts-ci" rel="noopener nofollow ugc">the template</a> should have read and write permission.<br>
Currently the template is broken. :cry:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/1/d108c2c99140f336ae21e79e19ddaf9563ad1442.png" data-download-href="/uploads/short-url/tPcyOaV1mOFOrq7B30L9f7zy5Qm.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/6702424/181099081-1f1f94ca-dbeb-495a-b7c6-4b9b04c4406e.png" alt="image" data-base62-sha1="tPcyOaV1mOFOrq7B30L9f7zy5Qm" width="573" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/1/d108c2c99140f336ae21e79e19ddaf9563ad1442_2_573x500.png, https://aws1.discourse-cdn.com/github/original/3X/d/1/d108c2c99140f336ae21e79e19ddaf9563ad1442.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/1/d108c2c99140f336ae21e79e19ddaf9563ad1442.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/1/d108c2c99140f336ae21e79e19ddaf9563ad1442_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">732×638 16.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/5/e53b1be3d4496b02a02b35bc0391b3f642c732bc.png" data-download-href="/uploads/short-url/wHRYPcXFStCrDQeplfV86jw5AtK.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/6702424/181099082-59769e60-0cb1-46f7-93b8-4a0d55757e6b.png" alt="image" data-base62-sha1="wHRYPcXFStCrDQeplfV86jw5AtK" width="690" height="143" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/5/e53b1be3d4496b02a02b35bc0391b3f642c732bc_2_690x143.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/5/e53b1be3d4496b02a02b35bc0391b3f642c732bc_2_1035x214.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/5/e53b1be3d4496b02a02b35bc0391b3f642c732bc_2_1380x286.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/5/e53b1be3d4496b02a02b35bc0391b3f642c732bc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1958×408 69 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks in advance</p>

**Comment by garronej** [comment, marked as answer]
<p>Fixed thank you github!</p>

**Comment by otter-computer** [comment]
<p>Hey <a class="mention" href="/garronej">@garronej</a>, thanks for reporting this, and for coming back to update the thread once it was fixed!</p>
<p>In future you may want to report issues like this in the <a href="https://github.com/github/feedback/discussions">feedback discussions</a> on <a href="http://github.com">github.com</a>. Giving feedback or reporting bugs in there puts it right in front of the engineering team so they’ll be able to tackle it quicker :sparkles:</p>
